parameter/ArrayParameter.cpp

```diff
@@ -135,13 +106,28 @@ bool CArrayParameter::accessValue(CPathNavigator& pathNavigator, string& strValu
 
         } else {
             // Scalar requested
-            doGetValue(strValue, getOffset() + index * getSize(), parameterAccessContext);
+            CParameter::doGetValue(strValue, getOffset() + index * getSize(), parameterAccessContext);
         }
     }
 
     return true;
 }
 
+/// Actual parameter access
+// String access
+bool CArrayParameter::doSetValue(const string& value, size_t /*offset*/,
+                                 CParameterAccessContext& parameterAccessContext) const
+{
+    return setValues(0, parameterAccessContext.getBaseOffset(), value, parameterAccessContext);
```

Comment:
I have no idea why offset is not used instead of `parameterAccessContext.getBaseOffset()`. If I forward offset, the regression tests are brocken. @benaware do know have an idea ? I must confess I do not understand all those offset manipulation.